addons/web/static/src/model/relational_model/relational_model.js

```diff
@@ -180,6 +181,18 @@ export class RelationalModel extends Model {
      */
     async load(params = {}) {
         const config = this._getNextConfig(this.config, params);
+        const groupByFieldName = config.groupBy?.[0]?.split(":")[0] || "";
```

Comment:
maybe move this in _loadGroupedList ? There, you know what you have a groupBy, and it's not empty

Reply:
better indeed, done